lyse/__init__.py

```diff
@@ -407,6 +407,22 @@ def get_trace(self,name):
             trace = h5_file['data']['traces'][name]
             return array(trace['t'],dtype=float),array(trace['values'],dtype=float)         
 
+    def get_waits(self):
+        """Return the time out status of waits.
+
+        Raises:
+            Exception: If the experiment has no waits.
+
+        Returns:
+            :obj:`numpy:numpy.ndarray`: Returns 2-D timetrace of times `'time'`
+            and time out status `'timed_out'`.
+        """
+        with h5py.File(self.h5_path,'r') as h5_file:
+            if not 'waits' in h5_file['data']:
+                raise Exception('The shot has no waits')
+            trace=h5_file['data']['waits']
+            return array((trace['time'],trace['timed_out']),dtype=float)  
```

Comment:
The `waits` object actually includes quite a bit more useful information than just the script start time and timeout state. Would you be opposed to returning all of the information (ie wait label, script start time, timeout time, actual length of the wait, and timeout state)?

Admittedly, that object is a [structured numpy array](https://numpy.org/doc/stable/user/basics.rec.html) where each column has a unique label and dtype, making the use a little different than normal numpy arrays. To get access to things, you'd need to use the label indexing.

Examples:
```python
waits = h5_file['data']['waits'][()]  # last bit is necessary to convert from h5py dataset to a numpy array
return waits
```
```python
waits.dtype  # [('label', 'S256'), ('time', '<f8'), ('timeout', '<f8'), ('duration', '<f8'), ('timed_out', '?')]
waits[0]  # gives the first wait object, note that printing this looks like a normal tuple
waits['timed_out']  # gives a boolean numpy array of the timed_out column
waits['label']  # gives a bytestring numpy array of the labels
```

Reply:
For my application I didn´t need other parameters, but it is no issue to return all the information.